src/enhanced/utils/queue/index.ts

```diff
@@ -13,7 +14,7 @@ export async function listenToJobQueue(): Promise<JobsQueuePayload> {
 
   const client = new SQS({ region });
 
-  console.log('Polling jobsQueue');
+  console.log('[listenToJobQueue]: Polling jobsQueue');
 
   // We want to loop indefinitely so that we continue to poll the queue.
   while (true) {
```

Comment:
Is there a limit on this loop? It seems that each possible error continues to iterate. Do we want a timebox or some other flag to end the process? I might have missed this somewhere.

Reply:
If an error occurs due to an improperly formatted message, we continue looping because that improperly formatted message will have been deleted from the queue. This means that the queue will no longer have the problematic message, and the task can continue polling for a valid job. 

Basically, if we get a bad message, we toss it out and wait until we get a good message. These errors _should_ be extremely rare, but these extra measures were thrown in just in case something went wrong somewhere.

Reply:
So, we have confidence that a good message will always be sent? Is there a time box we can set even if large so that we don't leave it running forever? Or is this not an issue and I'm not seeing the forest for the trees :)